src/state/Data/actions.js

```diff
@@ -8,24 +9,113 @@ import request from '../_common/request';
 import commonActions from '../_common/actions';
 
 import Select from "../Select";
-import {getMissingTiles} from './helpers';
+import {getMissingTiles, tileAsArray} from './helpers';
+import {TILED_LAYERS_TYPES} from './constants';
 
 const DEFAULT_RELATIONS_PAGE = {
     offset: 0,
     limit: 100,
 }
 
 /**
- * Check if given spatialData are defined and if contains data inside
- * @param {Object} spatialData Object recieved from BE contains under spatialDataKey object of data attributes [id]: {data, spatialIndex}. 
- * @return {bool}
+ * Centered place for getting PAGE_SIZE from state or configDefaults.
```

Comment:
```suggestion
 * Central method for getting PAGE_SIZE from state or configDefaults.
```

Reply:
This is before new config update of core, correct?

Reply:
Yes, the PR in ptr-core is still pending